src/core_modules/capture-core/rules/engine/converters/rulesEffectsValueConverter.js

```diff
@@ -70,9 +64,6 @@ class RulesValueConverter implements IConvertOutputRulesEffectsValue {
         return value;
     }
     convertAge(value: string): ?Object {
```

Comment:
```suggestion
    convertAge(value: string): Object {
```

Reply:
Looks to me that this will always return an object now. Perhaps this is unnecessary since `Object === any`